sycl/test-e2e/SubGroup/shuffle_fp16.cpp

```diff
@@ -4,6 +4,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+// UNSUPPORTED: hip
 // Even though `gfx908` and `gfx906` support halfs, libspirv is currently
```

Comment:
is this still relevant too?

Reply:
This is relevant. We don't have support for fp16 for AMDGPU yet